Makefile

```diff
@@ -15,14 +16,15 @@ help:
 	@echo "stop              Stops the Docker container."
 	@echo ""
 	@echo "Variables:"
-	@echo "SIMFILE_PATH       Path to the sim.json file."
+	@echo "SIMFILE_PATH      Path to the sim.json file."
```

Comment:
nit: this line and the one for `stop` three lines above end with a `.`, while the rest do not.